src/ui-components/cards/record-details-card.component.tsx

```diff
@@ -0,0 +1,27 @@
+import React from "react";
+import SummaryCard from "../cards/summary-card.component";
+import styles from "./record-details-card.css";
+
+export default function RecordDetails(props: DetailsProps) {
+  return (
+    <SummaryCard
+      name="Details"
```

Comment:
Should we translate this title?

Reply:
Yes, we should. I haven't found a reliable way yet to translate component prop strings that would work well with tests too. It appears things work better when I use the `Trans` component over the `useTranslation` hook. Regardless, I intend to have this resolved in my translations PR. I'm already using this `RecordDetails` component there.

Reply:
Ok, thanks!